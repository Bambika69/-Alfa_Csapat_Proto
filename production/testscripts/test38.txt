createfield
createvirologist F1
createvirologist F1
createagent virus bear V1
startturn V1
useagent BV1 V2