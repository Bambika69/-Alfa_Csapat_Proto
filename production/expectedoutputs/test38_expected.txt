Field created:
ID: F1
Type: Field
Neighbours: null

Virologist created:
ID: V1
Field: F1
Backpack: B1

Virologist created:
ID: V2
Field: F1
Backpack: B2

Agent created:
ID: A1
Type: virus
GeneticCode: bear
Virologist: V1
Result: Successful

Agent created:
ID: A2
Type: vaccine
GeneticCode: bear
Virologist: V2
Result: Successful

V2's turn started.

Agent used on virologist:
Agent: A2
Virologist: V2
Target: V2
Result: Failed

V1's turn started.

Agent used on virologist:
Agent: A1
Virologist: V1
Target: V2
Result: Failed
